WooCommerce/Classes/ViewRelated/Hub Menu/HubMenu.swift

```diff
@@ -31,6 +31,7 @@ struct HubMenu: View {
                     ForEach(viewModel.menuElements, id: \.self) { menu in
                         HubMenuElement(image: menu.icon, text: menu.title)
                             .frame(width: Constants.itemSize, height: Constants.itemSize)
+                            .contentShape(Rectangle())
```

Comment:
just curious, is this to increase the tappable area?

Reply:
Exactly. If you add a tap gesture to a primitive SwiftUI view such as Text or Image, the whole view becomes tappable. If you add a tap gesture to a container SwiftUI view, such as VStack or HStack, then SwiftUI only adds the gesture to the parts of the container that have something inside – large parts of the stack are likely to be untappable.  However, if you want to change the shape of hit tests – the area that responds to taps – then you should use the `contentShape()` modifier with the shape you want. 👍